Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -1312,6 +1313,7 @@ def test_http_requests_with_retry_negative_sanity(self, method, status):
                                       '',
                                       full_url='http://httpbin.org/status/{}'.format(status),
                                       retries=3,
+                                      backoff_factor=0.0001,
```

Comment:
```suggestion
```
If it's manual anyways I think it's best to keep the backoff as default to avoid manual running of the test from failing on overloading.